redbot/core/core_commands.py

```diff
@@ -1229,70 +1231,67 @@ async def perms(self, ctx, level: int):
         await ctx.send("The new permissions level has been set.")
 
     @commands.command()
-    @commands.guild_only()
     @checks.is_owner()
-    async def leave(self, ctx: commands.Context):
-        """Leaves the current server."""
-        await ctx.send(_("Are you sure you want me to leave this server? (y/n)"))
+    async def leave(self, ctx: commands.Context, *server_ids: GuildConverter):
+        """Leaves a server.
+
+        If no server IDs are passed the local server will be left instead."""
+        guilds = list(server_ids)
+        if ctx.guild is None and not guilds:
+            return await ctx.send(_("You need to specify at least one server ID."))
+
+        leaving_local_guild = not len(guilds) >= 1
+
+        if leaving_local_guild:
+            guilds.append(ctx.guild)
+            msg = _("You haven't passed any server ID. Do you want me to leave this server? (y/n)")
+        else:
+            msg = _("Are you sure you want me to leave {} servers? (y/n)".format(len(guilds)))
+
+        for guild in guilds:
+            if guild.owner.id == ctx.me.id:
+                return await ctx.send(
+                    _("I cannot leave the server `{}`: I am the owner of it.".format(guild.name))
+                )
 
+        await ctx.send(msg)
         pred = MessagePredicate.yes_or_no(ctx)
         try:
-            await self.bot.wait_for("message", check=pred)
+            await self.bot.wait_for("message", check=pred, timeout=30)
         except asyncio.TimeoutError:
             await ctx.send(_("Response timed out."))
             return
         else:
             if pred.result is True:
-                await ctx.send(_("Alright. Bye :wave:"))
-                log.debug(_("Leaving guild '{}'").format(ctx.guild.name))
-                await ctx.guild.leave()
+                if leaving_local_guild is True:
+                    await ctx.send(_("Alright. Bye :wave:"))
+                else:
+                    await ctx.send(_("Alright. Leaving {} servers...".format(len(guilds))))
+                for guild in guilds:
+                    log.debug(_("Leaving guild '{}'").format(guild.name))
```

Comment:
Logs shouldn't be translated (I know this is as it was before the PR but if we're changing this whole part of code, we might as well fix this :))
```suggestion
                    log.debug("Leaving guild '%s' (%s)", guild.name, guild.id)
```